static/js/ui_init.js

```diff
@@ -138,6 +139,14 @@ function initialize_left_sidebar() {
     $("#left-sidebar-holder").html(rendered_sidebar);
 }
 
+function initialize_right_sidebar() {
+    const rendered_sidebar = render_right_sidebar({
+        can_invite_others_to_realm: page_params.show_invites,
```

Comment:
Changed this to `settings_data.user_can_invite_others_to_realm()`, since we can compute this on the frontend.

This may mean we get to remove `compute_show_invites` as well in a follow-up PR.